views/package.json

```diff
@@ -9,6 +9,6 @@
   "license": "GPL-2.0",
   "dependencies": {
     "@oat-sa/tao-test-runner": "0.5.0",
-    "@oat-sa/tao-test-runner-qti": "2.13.6"
+    "@oat-sa/tao-test-runner-qti": "oat-sa/tao-test-runner-qti-fe#feature/TBD-46/stop-sending-csrf-tokens-to-get-item-request-handlers"
```

Comment:
⚠️ To be replaced once a stable version will have been released.

Reply:
2.15.0 released